apps/docs/docs/modules/llms/available_llms/groq.md

```diff
@@ -0,0 +1,83 @@
+# Groq
+
+## Usage
+
+```ts
+import { Groq, serviceContextFromDefaults } from "llamaindex";
+
+const groq = new Groq({
+  apiKey: "<YOUR_API_KEY>",
+});
+
+const serviceContext = serviceContextFromDefaults({ llm: groq });
+```
+
+## Load and index documents
+
+For this example, we will use a single document. In a real-world scenario, you would have multiple documents to index.
```

Comment:
@gradenr Great that you added docs. You can even move the examples to the `/examples` folder and reference them in your doc file, see https://github.com/run-llama/LlamaIndexTS/blob/2ec6a529c7d7d9f1fe9aff38f16568bdd3e27072/apps/docs/docs/modules/data_loader.mdx#L5 for how to do that

Reply:
I moved the "full example" to examples and loaded it with code blocks.  I'm not so sure what to do with the smaller snippets.

Reply:
That's fine thanks!